pages/common/fortune.md

```diff
@@ -0,0 +1,27 @@
+# fortune
+
+> Print a random quotation (fortune-cookie style)
+
+- Print a quotation
+
+`fortune`
+
+- Print a quotation from a given database
```

Comment:
The database is really just a file, right? If so I think it would be better to refer to it as such in this example.